rust/src/lib/query_apply/route.rs

```diff
@@ -74,16 +74,17 @@ impl MergedRoutes {
         ignored_ifaces: &[&str],
         current_ifaces: &Interfaces,
     ) -> Result<(), NmstateError> {
-        let mut cur_routes: Vec<&RouteEntry> = Vec::new();
-        if let Some(cur_rts) = current.config.as_ref() {
-            for cur_rt in cur_rts {
+        let mut cur_routes: Vec<RouteEntry> = Vec::new();
+        if let Some(cur_rts) = current.config.clone() {
+            for mut cur_rt in cur_rts {
                 if let Some(via) = cur_rt.next_hop_iface.as_ref() {
                     if ignored_ifaces.contains(&via.as_str())
                         && cur_rt.route_type.is_none()
                     {
                         continue;
                     }
                 }
+                cur_rt.sanitize_current_route_for_verify();
```

Comment:
Why is this needed for quickack and not for other options?

Reply:
Because no other route property is bool.

Reply:
Can you expand a bit more? How does that affect us?

Reply:
Can you add a comment explaining it?
Apart from that, LGTM and can be merged.

Reply:
Comment added.